src/data.js

```diff
@@ -5824,6 +5823,28 @@ module.exports = [
     phone: 'iphone',
     tags: ['Full Stack', 'Developer', 'Front End', 'JavaScript', 'React'],
   },
+  {
+    name: 'Jorge Ruvalcaba',
+    description:
+      'Software Engineer, Senior Frontend Engineer, Taco Master and internet weirdo 🤓',
+    url: 'https://jorgeruvalcaba.dev/uses/',
```

Comment:
i only get

> Not Found
> You just hit a route that doesn't exist... the sadness.

Reply:
My bad 🤦  . Didn't push the page before but now it's alive. You can check it out [here](https://jorgeruvalcaba.dev/uses/).